content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -85,13 +86,14 @@ dvc.lock  training_metrics       train.py
 dvc.yaml  training_metrics.json
 ```
 
-The [metrics history](/doc/dvclive/get-started#history) generated under
-`training_metrics` can be visualized with `dvc plots`.
-
-In addition, the [metrics summary](/doc/dvclive/get-started#summary)
```

Comment:
👍 There's another metrics summary link under `--live-no-summary` that needs to be updated.